src/event/event_epoll.c

```diff
@@ -89,12 +89,12 @@ _dispatch_muxnote_bucket(int ident)
 	return &_dispatch_sources[DSL_HASH((uint32_t)ident)];
 }
 #define _dispatch_unote_muxnote_bucket(du) \
-	_dispatch_muxnote_bucket(du._du->du_ident)
+	_dispatch_muxnote_bucket((int)du._du->du_ident)
 
 DISPATCH_ALWAYS_INLINE
 static inline dispatch_muxnote_t
 _dispatch_muxnote_find(struct dispatch_muxnote_bucket_s *dmb,
-		uint64_t ident, int16_t filter)
+		int ident, int16_t filter)
```

Comment:
we need a cast somewhere on this call path because dmn_ident is an int (makes since since file descriptors are ints), but du_ident is an uint32_t.   

I can push the cast down to the == compare instead of bubbling it up.